index.css

```diff
@@ -36,6 +36,13 @@ nav h1 a {
   padding-right: 14%;
 }
 
+.fa-close {
+  padding-right: 7%;
+  font-size: 24px;
+  float: right;
+  padding-top: 10px;
+}
```

Comment:
@levy002
Thanks for the feedback.
But how can i generate this same color using rgba because otherwise its very faint?
Ill appreciate any help rendered